spcs-basic/build-and-push.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+DB_NAME="spcs_na_db"
+SCHEMA_NAME="public"
+IMAGE_REPO_NAME="images"
```

Comment:
We should mention that both `manifest.yml` and `service_spec.yml` need to agree with these + image name.